spec/order_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'order'
+
+describe Order do
+
+  let(:order) { described_class.new }
+  let(:dish1) { double(:dish1, name: "Dishy1", price: 3.50) }
+  let(:dish2) { double(:dish2, name: "Dishy2", price: 5.00) }
+  let(:dish3) { double(:dish3, name: "Dishy3", price: 4.99) }
+  let(:dish4) { double(:dish4, name: "Dishy4", price: 5.99) }
+  let(:restaurant1) { double(:restaurant1, select: dish1) }
+  let(:restaurant2) { double(:restaurant2, select: dish2) }
+  let(:restaurant3) { double(:restaurant3, select: dish3) }
+  let(:restaurant4) { double(:restaurant4, select: dish4) }
+
+
+  describe '#add_to_basket' do
+    it 'calls the menu#select method' do
+      expect(restaurant1).to receive(:select)
+      order.add_to_basket(restaurant1, 1, 2)
+    end
+
+    it 'given the menu, dish number and quantity adds
+        the dish/dishes to the order' do
+      order.add_to_basket(restaurant1, 1, 2)
+      expect( (order.basket[dish1] / dish1.price).to_i ).to eq 2
+    end
+  end
+
+  describe '#summary' do
+    it 'gives a summary of the order, grouping
+        items and displaying prices with a total' do
+      order.add_to_basket(restaurant1, 1, 2)
+      order.add_to_basket(restaurant2, 1, 3)
+      expected_output = "2x Dishy1 | £7.00\n" +
```

Comment:
Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.